res/css/views/auth/_AuthBody.scss

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 */
 
 .mx_AuthBody {
-    width: 500px;
+    max-width: 500px;
```

Comment:
Hmm, for large screens at least, we don't want the width to change based on content. Could we leave this as it, and then reset width to auto and use max-width for small screens?